frontend/packages/console-app/console-extensions.json

```
@@ -1090,6 +1308,55 @@
       }
     }
   },
+  {
+    "type": "console.navigation/resource-ns",
+    "properties": {
+      "perspective": "admin",
+      "section": "storage",
+      "id": "volumegroupsnapshots",
+      "insertAfter": "volumesnapshotcontents",
+      "name": "%console-app~VolumeGroupSnapshots%",
+      "model": {
+        "group": "groupsnapshot.storage.k8s.io",
+        "version": "v1beta1",
+        "kind": "VolumeGroupSnapshot"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-cluster",
+    "properties": {
+      "perspective": "admin",
+      "section": "storage",
+      "id": "volumegroupsnapshotclasses",
+      "insertAfter": "volumegroupsnapshots",
+      "name": "%console-app~VolumeGroupSnapshotClasses%",
+      "model": {
+        "group": "groupsnapshot.storage.k8s.io",
+        "version": "v1beta1",
+        "kind": "VolumeGroupSnapshotClass"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-cluster",
+    "properties": {
+      "id": "volumegroupsnapshotcontents",
```

we don't need `perspective` here ??
```suggestion
      "id": "volumegroupsnapshotcontents",
      "perspective": "admin",
```